larch.js

```diff
@@ -90,6 +92,21 @@ function logMultiBackend(level, msg, meta, cb) {
     }
 };
 
+Larch.prototype.willSampleSingleBackend =
+function willSampleSingleBackend(level) {
+    return self.backends[0].willSample(level);
+};
+
+Larch.prototype.willSampleMultiBackend =
+function willSampleMultiBackend(level) {
```

Comment:
Consider: I think for optimum performance we want to ensure that ALL backends sample and not log if a single backend cannot sample.

I think we should set our sampling thresholds tailored towards "if this is hit just dont log to any backend".